exec/java-exec/src/main/java/org/apache/drill/exec/record/metadata/AbstractColumnMetadata.java

```diff
@@ -30,10 +30,11 @@
 import org.apache.drill.exec.record.metadata.schema.parser.SchemaExprParser;
 import org.joda.time.format.DateTimeFormatter;
 
-import java.io.IOException;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.stream.Collectors;
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
```

Comment:
Is this correct import order?

Reply:
It is the order into which Eclipse sorts the imports automatically on save. If Drill has a preferred order, I can see if I can get Eclipse to follow that order; or I can turn off the auto-sort and organize them manually.